src/openmrs-esm-login.d.tsx

```diff
@@ -2,9 +2,4 @@
 // for @openmrs/esm-api published to npm
 declare type Observable<T> = import("rxjs").Observable<T>;
```

Comment:
nit: do you mind renaming this to declarations.d.tsx just like you did the other modules? :-)

Reply:
👍 updated